lib/point_quest/quests/quest.ex

```diff
@@ -115,20 +115,20 @@ defmodule PointQuest.Quests.Quest do
     }
   end
 
-  def project(%Event.RoundStarted{quest_objective: objective}, %__MODULE__{} = quest) do
+  def project(%Event.RoundStarted{objectives: objectives}, %__MODULE__{} = quest) do
```

Comment:
Do we need to pass all objectives into a round event because this is how the sort order can change and then update the quest? I am surprised seeing all the rounds as a value from the event.

Reply:
Not sure if I'm misunderstanding but currently we don't tie the objective to the round other than that a given objective is "current" as a result of the start_round command

Reply:
Happy to discuss alternatives!